modules/virt-creating-custom-resources-hpp.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-custom-resources-hpp_{context}"]
+= Creating host path provisioner (HPP) custom resources
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, you are not required to change your HPP custom resource. You continue using your existing storage classes. You do not have to modify the HPP custom resource for the legacy provisioner DaemonSet.
+
+== Creating the legacy HPP custom resource
+
+If you are installing {VirtProductName} version 4.10 on a new cluster, you must create an HPP legacy custom resource:
+
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+spec:
+  imagePullPolicy: IfNotPresent
+  pathConfig:
+    path: "/var/hpvolumes"
+    useNamingPrefix: false
+ workload:
+   nodeSelector:
+     kubernetes.io/os: linux
+----
+
+== Creating the CSI driver provisioner custom resource
+
+The legacy provisioner DaemonSet and the CSI driver DaemonSet are supported in parallel for a number of releases. However, at some point, the legacy provisioner DaemonSet will no longer be supported. To prepare for this change,
```

Comment:
I would eliminate the use of the word Daemonset from the entire document. Its an implementation detail, not a configuration detail.

Reply:
I exorcised the DAEMONs :-)